02 - Community Expansions/02.01 Plugins by Category/Plugins to manage internal and external links.md

```diff
@@ -7,15 +7,14 @@ publish: true
 ---
 
 
-# Link
+# Plugins to manage internal and external links
 
-Edit external or internal links
+Plugins to edit external or internal links.
```

Comment:
some of those (e.g. map of content plugin) do not "edit" links in any way though, but only display them in a different way than native Obsidian.